### ISSUE TITLE
Some elements on page when open in "New wavebox window" does not rerender

### ISSUE THREAD

**Issue by nmat** [wavebox/waveboxapp#1000]
* Wavebox Version: 4.9.1
* Operating System & Version: Windows 10
* Account type (if applicable): Jira

**Expected & actual behavior**
Scale content based on resize of window

**Steps to reproduce**
(not tested thoroughly):
1. Open jira service desk
2. Open a link as "New wavebox window"
3. Resize the window

Some of the content is not resizes based on window resize. These do look like normal divs with some javascript connected to them. 

**Is the bug persistent or intermittent?**
Persistent

**Additional information**
I have only reproduced it on one system. However I have reinstalled wavebox from scratch and it seems that there can be elements on a page that is not resized on window resize.

I tested slack also but that one resizes information as intended. So it might be something specific on jira page that is not being resized correctly.

If I Reload the page after resize, then the elements are sized to the window borders though. So some background resizing does not seem to follow on window resize.

**Comment by Thomas101** [comment]
Is this on a machine with one hidpi monitor? I'm wondering if it's something to do with that and possibly related to #634 and #604 ?

What kind of link did you open up? I tried this one and it seemed to be okay...
![Screenshot 2019-04-02 at 09 51 37](https://user-images.githubusercontent.com/103586/55389389-08bcf000-552d-11e9-8406-ac884024b517.png)

**Comment by nmat** [comment]
yes that is correct. hidpi and both issues mentioned are the same as I see here.

It however seems odd I have to reload _every_ time I resize the window.

I open a ticket using Right click -> open in wavebox -> new window then tried resizing the window after moving it from the laptop to external display.

**Comment by Thomas101** [comment]
It sounds like it could be a dpi issue. I've just given it a try on macOS, but will also double check on a windows machine when I get back to one.

Was there a previous version of the app this was working with or has it always been like this?

**Comment by Thomas101** [comment]
Okay, I've given it a try on a Windows machine with one regular dpi monitor and one hidpi monitor and don't see the problem. It could be down to some layout setup that I'm doing differently,

* When you open the new window, does it open on the same monitor as the main window?
* Have you snapped any of the windows (.e.g drag to the left/right of the screen)?
* Have you maximised of fullscreened any of the windows?

**Comment by nmat** [comment]
On the laptop (hidpi) I have wavebox full screened.

Right clicked on a link in jira service desk and then dragged the window to a non hidpi.

Normal resize the window and the content is stuck.

Reload the content and it resizes.

Resize the window down and the content is stuck again.

Monitor is connected with a USB-C connector and is a widescreen monitor.

**Comment by Thomas101** [comment]
I've tried the same here, with all combinations of...

* Main window maximized / fullscreen / just sized
* Primary monitor set to hidpi and non hidpi
* Main window on hidpi and non hidpi

I'm a little bit out of ideas here 🤔. I wonder if it could be some kind of driver issue - although that feels like blaming it on the wind.

**Comment by nmat** [comment]
I am going to see if I can create a good test case for it. The effect is rather the same as I had with starting game and going back to the window of a open file.

Leave this as is for now and I will see if I can produce something for doing testing.
